actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -249,7 +205,8 @@ def tag(name = nil, options = nil, open = false, escape = true)
         if name.nil?
           tag_builder
         else
-          "<#{name}#{tag_builder.tag_options(options, escape) if options}#{open ? ">" : " />"}".html_safe
+          attributes = tag_attributes(options, escape) if options
+          "<#{name}#{attributes ? " #{attributes}" : nil}#{open ? ">" : " />"}".html_safe
```

Comment:
Should `tag_attributes_with_leading_space` be defined on `ActionView::Helpers::TagHelper` (instead of `ActionView::Helpers::TagHelper::TagBuilder`) so it can be used here?

Reply:
Wouldn't that make it a public method? I've instead made it public on TagBuilder which we can use here. What do you think?

Reply:
I like that even better. :+1: